src/main/java/Player_Types/Player

```diff
@@ -20,29 +22,56 @@ public class Player implements Death{
         this.attackDamage = attackDamage;
         this.damageMultiplier = damageMultiplier;
         this.money = money;
-        this.XP = XP;
+        this.currEXP = XP;
+        this.maxEXP = 10;
         this.inventory = inventory;
     }
-
-    public void increaseMoney(int numberToIncrease){
-        this.money = this.money + numberToIncrease;
-    }
-
-    public void decreaseMoney(int numberToDecrease){
-        this.money = this.money + numberToDecrease;
+    public Player(String name, int HP, int attackDamage, int damageMultiplier, int money, int XP) {
+        this.name = name;
+        this.HP = HP;
+        this.attackDamage = attackDamage;
+        this.damageMultiplier = damageMultiplier;
+        this.money = money;
+        this.currEXP = XP;
+        this.maxEXP = 10;
+        this.inventory = new ArrayList<>();
     }
 
+    @Override
     public String getName() {
         return name;
     }
-
-    public void setName(String name) {
-        this.name = name;
+    @Override
```

Comment:
I like the fact that all the manipulation is done through an interface, rather than within the class itself as this adheres to the clean architecture.